joint_trajectory_controller/include/joint_trajectory_controller/hardware_interface_adapter.h

```diff
@@ -213,7 +216,7 @@ class HardwareInterfaceAdapter<hardware_interface::VelocityJointInterface, State
     // Update PIDs
     for (unsigned int i = 0; i < n_joints; ++i)
     {
-      const double command = pids_[i]->computeCommand(state_error.position[i], state_error.velocity[i], period);
+      const double command = (desired_state.velocity[i] * use_velocity_ff_) + pids_[i]->computeCommand(state_error.position[i], state_error.velocity[i], period);
```

Comment:
Me neither. I had always assumed that the feedforward was either there or not, but let's assume whoever put the `0.9` in the COB knew what he was doing and make the parameter a double.

Then, what would you suggest for values greater than `1.0`? Should we raise some warning and clamp, just raise a warning or do nothing altogether?